packages/docusaurus-plugin-google-gtag/src/types.d.ts

```diff
@@ -6,15 +6,3 @@
  */
 
 /// <reference types="@docusaurus/module-type-aliases" />
-
-interface Window {
-  gtag: (
-    command: string,
-    fields: string,
-    params: {
-      page_title?: string;
-      page_location?: string;
-      page_path?: string;
-    },
-  ) => void;
-}
```

Comment:
note: this type is only used internally, and is not exposed by the gtag package, so it shouldn't affect anyone.